utils/unittest/logging.go

```diff
@@ -19,13 +19,13 @@ func LogVerbose() {
 // Logger returns a zerolog
 // use -vv flag to print debugging logs for tests
 func Logger() zerolog.Logger {
-	writer := ioutil.Discard
+	return LoggerWithLevel(zerolog.DebugLevel)
```

Comment:
⚠️  This is pretty significantly changing the behaviour of `Logger` for all other usages. Previously it would only log when you set the `verbose` flag, now it will log to the test output (even if the test is passing - which isn't what we want for more cases). 

I think we should keep the default behaviour, and add a method which allows specifying the output directly for the integration tests, if that's the behaviour we want there.